test/integration/fixtures/options/bail.fixture.js

```diff
@@ -13,11 +13,11 @@ describe('suite1', function () {
 });
 
 describe('suite2', function () {
-  before(function (done) {
+  before(function () {
```

Comment:
was this causing a problem?  perhaps your changes should actually be additions; two tests that use `done` and two that don't.  this would give us confidence that the same thing works properly regardless of the Runnable's strategy (async or sync)

Reply:
no, no problem arose, the 'suite2' describe is skipped (correctly) anyway. I added a second test, one sync and one async.